gateway-js/src/config.ts

```diff
@@ -325,6 +326,7 @@ export function isManagedConfig(
     'schemaConfigDeliveryEndpoint' in config ||
     'uplinkEndpoints' in config ||
     'fallbackPollIntervalInMs' in config ||
+    (isSupergraphManagerConfig(config) && config.supergraphSdl instanceof UplinkSupergraphManager) ||
```

Comment:
**note** This change is needed so we can treat a user-constructed `UplinkSupergraphManager` as a managed config when deciding how to behave (i.e. implicit managed config should be the same as passing in your own `UplinkSupergraphManager`)